coalib/bearlib/__init__.py

```diff
@@ -82,34 +100,56 @@ def _deprecate_decorator(func):
         logged_deprecated_args = set()
 
         def wrapping_function(*args, **kwargs):
-            for arg, depr_arg_and_modifier in depr_args.items():
-                deprecated_arg, _func = (
```

Comment:
@jayvdb is right. You're getting an arg_tuple here just to rename it to depr_args_and_modifier again and never use arg_tuple. That's unneeded complexity.